task.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# создаём каталог task с вложенными директориями
+# task
+#   dir1
+#   dir2
+#   dir3
+#       dir4
+mkdir -p task/dir1 task/dir2 task/dir3 task/dir3/dir4
```

Comment:
Так как есть флаг `-p`, то создание папки3 можно опустить, так как она есть в последнем аргументе
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```